packages/governance/test/swingsetTests/contractGovernor/bootstrap.js

```diff
@@ -14,7 +14,7 @@ const { quote: q } = assert;
  * @param {ERef<ZoeService>} zoe
  * @param {(string:string) => undefined} log
  * @param {Record<string,Installation>} installations
- * @param {ERef<GovernedContractFacetAccess<unknown>>} contractFacetAccess
+ * @param {ERef<GovernedContractFacetAccess<{},{}>>} contractFacetAccess
```

Comment:
I think `unknown` would be more accurate than the empty object, but if that opens a can of worms nevermind :)